Makefile

```diff
@@ -0,0 +1,68 @@
+SHELL = /bin/bash
+REG = quay.io
+ORG = integreatly
+IMAGE = apicurio-operator
+TAG = latest
+KUBE_CMD = oc apply -f
+RESOURCES_DIR = ./res
+DEPLOY_DIR = deploy
+OUT_STATIC_DIR = build/_output
+OUTPUT_BIN_NAME = ${IMAGE}
+TARGET_BIN = cmd/manager/main.go
+NS = apicurio-operator-test
+SA = apicurio-operator
+
+travis/setup:
+	@echo Installing golang dependencies
+	@go get golang.org/x/sys/unix
+	@go get golang.org/x/crypto/ssh/terminal
+	@echo Installing dep
+	curl https://raw.githubusercontent.com/golang/dep/master/install.sh | sh
+	@echo Installing errcheck
+	@go get github.com/kisielk/errcheck
+	@echo setup complete run make build deploy to build and deploy the operator to a local cluster
+	dep ensure
+
+code/k8s:
+	operator-sdk generate k8s
+
+code/fix:
+	gofmt -w `find . -type f -name '*.go' -not -path "./vendor/*"`
+
+code/check:
+	diff -u <(echo -n) <(gofmt -d `find . -type f -name '*.go' -not -path "./vendor/*"`)
+
+code/compile:
+	go build -o ${OUTPUT_BIN_NAME} ${TARGET_BIN}
+
+test/unit:
+	go test -v -race -cover ./pkg/...
+
+test/e2e/cluster:
+	operator-sdk test cluster --namespace ${NS} --service-account ${SA} ${REG}/${ORG}/${IMAGE}:${TAG}
```

Comment:
Is there an advantage to passing the namespace here other than knowing what namespace it will be running in? How does that work for parallel testing?